components/StyledButton.js

```diff
@@ -0,0 +1,16 @@
+import styled, { css } from "styled-components";
```

Comment:
The `css` import is not needed here because we're not using `css` helper functions in this component. 
```suggestion
import styled from "styled-components";
```